src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -164,17 +165,20 @@ internal SentryUnityOptions(IApplication application, bool isBuilding)
             StackTraceMode = StackTraceMode.Original;
             IsEnvironmentUser = false;
 
+            var releaseSB = new StringBuilder();
```

Comment:
I'm curious: Why did you go with a StringBuilder here?

Reply:
habit, but it's an overkill here, reverting to string concatenation